robot/description/urdf/stand_link.urdf.xacro

```diff
@@ -6,8 +6,32 @@
     
     <!-- Stand link; this is a dummy link that is used to make camera placement and other sensors on the stand more convenient -->
     <!-- Stand link definition. Note: purposely left blank -->
-    <link name="${tf_prefix}/stand_link">
+    <link name="${tf_prefix}/stand_link"/> 
+
+    <!-- ZED Camera -->
+    <link name="${tf_prefix}/zed_camera_link">
+        <visual>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <geometry>
+            <box size="${ZED_camera_width} ${ZED_camera_length} ${ZED_camera_height}"/>
+            </geometry>
+        </visual>  
+        <collision>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <geometry>
+                <box size="${ZED_camera_width} ${ZED_camera_length} ${ZED_camera_height}"/>
+            </geometry>
+        </collision>
+        <inertial>
+            <xacro:box_inertia mass="${ZED_camera_mass}" length="${ZED_camera_length}" width="${ZED_camera_width}" height="${ZED_camera_height}"/>
+            <mass value="${ZED_camera_mass}"/>
+        </inertial>
     </link>
+    <joint name="${tf_prefix}/zed_camera_joint" type="fixed">
+        <origin xyz="${stand_length/2 - ZED_camera_width} 0 ${ZED_camera_height / 2.0}" rpy="0 0 0"/>
```

Comment:
For the x position, do you need to do all this math or can you just set it to a constant called stand_x_position or something (refer to other constants for naming convention) ?

Reply:
I think its accurate, so whynot

Reply:
If we were to use `stand_x_position`, I would still define the `stand_length` variable and then set `stand_x_position = stand_length / 2 - ZED_camera_width`. But having an extra `stand_x_position` variable seems excessive, given how *obvious* it is where we are placing the ZED stereo cam

Reply:
Also I believe the origin of the ZED stereo cam is in its center so you would need to use `ZED_camera_width / 2.0` so that the whole stereo cam is on the the plate, right at the edge